src/components-pages/Error.jsx

```diff
@@ -1,5 +1,8 @@
-export const Error = () => {
-    return (<div> 
-    <p>Page Not Found</p>
+export const Error = (params) => {
+    const {errorStatus, errorMessage} = params
+    return (<div className="error-page"> 
+    <h1>{errorStatus} error</h1>
```

Comment:
I wouldn't use `<h1>` here as it's not best practice and I definitely wouldn't use it twice in a row, I would stick to having your main header as `<h1>` and go down in heading levels, the heading number doesn't denote a size as you can set this however big you want in your CSS